bundles/org.openhab.binding.bmwconnecteddrive/src/main/resources/OH-INF/thing/vehicle-status-channel-types.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="bmwconnecteddrive"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="doors-channel">
+		<item-type>String</item-type>
+		<label>Overall Door Status</label>
+	</channel-type>
+	<channel-type id="windows-channel">
+		<item-type>String</item-type>
+		<label>Overall Window Status</label>
+	</channel-type>
+	<channel-type id="lock-channel">
+		<item-type>String</item-type>
+		<label>Doors Locked </label>
+	</channel-type>
+	<channel-type id="next-service-date-channel">
+		<item-type>DateTime</item-type>
+		<label>Next Service Date</label>
+	</channel-type>
+	<channel-type id="next-service-mileage-channel">
+		<item-type>Number:Length</item-type>
+		<label>Mileage till Next Service</label>
```

Comment:
The first letter of every word in a label must be capitalized.
```suggestion
		<label>Mileage Till Next Service</label>
```

Reply:
fixed